Documentation/compatibility/ASPNET Accessibility Improvement.md

```diff
@@ -0,0 +1,53 @@
+
+ Quirked
+
+ Build-time breake## ASP.NET Accessibility Improvement in .NET 4.7.1
+
+### Scope
+Minor
+
+
+### Version Introduced
+.NET 4.7.1
+
+
+### Source Analyzer Status
+NotPlanned
+
+
+### Change Description
+ASP.NET is improving how ASP.NET Web Controls work with accessibility technology in Visual Studio to better support ASP.NET customers.  These include the following changes:
+- Changes to implement missing UI accessbility patterns in controls, like the Add Field dialog in the Details View wizard.
+- Changes to improve the display in High Contrast mode, like the Data Pager Fields Editor.
+- Changes to improve the keyboard navigation experiences for controls, like the Configure Object Context Window or the Configure Data Source Window.
+
+- [x] Quirked 
+- [ ] Build-time break 
+
+
+### Recommended Action
++__How to opt in or out of these changes__
+
++In order for the Visual Studio Designer to benefit from these changes, it must run on the .NET Framework 4.7.1 or later. The web application can benefit from these changes in either of the following ways:
```

Comment:
The two lines above shouldn't start with a `+`.

Reply:
Will be fixed in the next commit.